modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -590,6 +590,35 @@ definition {
 			title = "Numeric Field - Table View Title");
 	}
 
+	@description = "Automate LPS-103467 Define object's fields at Form View Level"
+	@priority = "5"
+	test CheckRequiredOption {
+		property portal.acceptance = "true";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+
+		FormViewBuilder.setFieldLabel(fieldLabel = "Name");
+
+		FormViewBuilder.validateDisabledRequiredField();
```

Comment:
Hi Calvin, here you have to validate that the kebab menu from "Required Field" is disabled, not the switch button.